compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/ComposeFoundationFlags.kt

```diff
@@ -77,7 +77,7 @@ object ComposeFoundationFlags {
      * [BasicTextField][androidx.compose.foundation.text.BasicTextField]s. If false, the previous
      * context menu that has no public APIs will be used instead.
      */
-    @Suppress("MutableBareField") @JvmField var isNewContextMenuEnabled = true
+    @Suppress("MutableBareField") @JvmField var isNewContextMenuEnabled = false
```

Comment:
can we make default value expect/actual instead? It will allow to enable it per platform during implementation

Reply:
Also please add explicit TODOs with links to tasks

Reply:
> can we make default value expect/actual instead? It will allow to enable it per platform during implementation

Sure, but not in this PR.


> Also please add explicit TODOs with links to tasks

Sure, done.